packages/application-components/package.json

```diff
@@ -30,6 +30,9 @@
     "@babel/runtime": "^7.22.15",
     "@babel/runtime-corejs3": "^7.22.15",
     "@commercetools-frontend/actions-global": "22.8.1",
+    "@commercetools-frontend/application-config": "22.8.1",
+    "@commercetools-frontend/application-shell": "22.8.1",
```

Comment:
⚠️ This introduces a circular dependency. What is required from app shell in the components?

Reply:
Yes. This is something I already noted to discuss in our next sync meeting.